markdown/ch3.markdown

```diff
@@ -345,17 +338,19 @@ plt.plot(sdsig);
 Although it still looks wobbly, the *signal-to-noise ratio* (SNR),
 is much greater in this version than when using the simple difference filter.
 
-(Note: this operation is called filtering because, in physical electrical
-circuits, many of these operations are implemented by hardware that
-lets certain kinds of current through, but not others; these components
-are called filters. For example, a common filter that removes high-frequency
-voltage fluctuations from a current is called a *low-pass filter*.)
+> **A note about filtering** {.callout}
+>
+> This operation is called filtering because, in physical electrical
+> circuits, many of these operations are implemented by hardware that
+> lets certain kinds of current through, but not others; these components
+> are called filters. For example, a common filter that removes high-frequency
+> voltage fluctuations from a current is called a *low-pass filter*.
 
 ## Filtering images (2D filters)
 
-Now that you've seen filtering in 1D, I hope you'll find it straightforward
-to extend these concepts to 2D. Here's a 2D difference filter finding the
-edges in the coins image:
+Now that you've seen filtering in 1D, we hope you'll find it straightforward to
+extend these concepts to 2D signals, a.k.a. images. Here's a 2D difference
```

Comment:
I wouldn't call anything other than a ligh-capture an image typically, even though we do, in practice, include a wider variety of data.  But, e.g., voltage measured on a 2D surface is probably not an image.